packages/backend/src/index.ts

```diff
@@ -491,6 +493,36 @@ export function initIocContainer(
   return container
 }
 
+export const parseAndLookupAddresses = async (
+  replicaAddresses: string[]
+): Promise<string[]> => {
+  const parsed = []
+  for (const addr of replicaAddresses) {
+    const parts = addr.split(':')
+    if (!parts)
+      throw new Error(
+        `Cannot parse replicaAddresses in 'accountingService' startup - value: "${addr}"`
+      )
+    if (parts.length === 1 && !isNaN(Number(parts[0]))) {
+      parsed.push(parts[0])
+      continue
+    }
+
+    if (net.isIP(parts[0]) === 0) {
+      try {
+        await dns.promises.lookup(parts[0], { family: 4 }).then((resp) => {
+          parsed.push(`${resp.address}:${parts[1]}`)
+        })
```

Comment:
```suggestion
        const resp = await dns.promises.lookup(parts[0], { family: 4 })
        parsed.push(`${resp.address}:${parts[1]}`)
```
I'd have to check but I don't remember what happens when doing await with `.then` and no `.catch` handler. Probably ok but this is more straightforward